packages/cli/src/prompts.ts

```diff
@@ -16,17 +16,27 @@
 import chalk from 'chalk'
 import moment from 'moment'
 import os from 'os'
+import _ from 'lodash'
+
+export const deployOrValidate = (checkOnly: boolean, capitalize: boolean, suffix: boolean): string => {
```

Comment:
Can we use named paramters here, I think the calling to this function is a bit unreadable otherwise. Also I think suffix is confusing, what you actually mean is noun
```suggestion
export const deployOrValidate = ({ checkOnly, capatilize, noun }: { checkOnly: boolean; capitalize: boolean; noun: boolean }): string => {
```